app/scenes/channel_info/channel_info_container.js

```diff
@@ -3,15 +3,26 @@
 
 import {bindActionCreators} from 'redux';
 
-import navigationSceneConnect from '../navigationSceneConnect';
-
 import {goToChannelMembers, goToChannelAddMembers, goBack} from 'app/actions/navigation';
-import {getChannelStats, deleteChannel} from 'mattermost-redux/actions/channels';
-import {markFavorite, unmarkFavorite, leaveChannel} from 'app/actions/views/channel';
-import {getCurrentChannel, getCurrentChannelStats, getChannelsByCategory, canManageChannelMembers} from 'mattermost-redux/selectors/entities/channels';
+import {
+    closeDMChannel,
+    closeGMChannel,
+    leaveChannel,
+    markFavorite,
+    unmarkFavorite
+} from 'app/actions/views/channel';
 import {getTheme} from 'app/selectors/preferences';
+
+import {getChannelStats, deleteChannel} from 'mattermost-redux/actions/channels';
+import {
+    getCurrentChannel,
+    getCurrentChannelStats,
+    getChannelsByCategory,
+    canManageChannelMembers
+} from 'mattermost-redux/selectors/entities/channels';
 import {getUser} from 'mattermost-redux/selectors/entities/users';
 
+import navigationSceneConnect from '../navigationSceneConnect';
```

Comment:
Could you change this to not be relative? I'd like to avoid relative imports that go backwards